infra/k8s/testground-infra/values.yaml

```diff
@@ -132,6 +132,12 @@ redis:
       - name: net.netfilter.nf_conntrack_max
         value: "100000"
   master:
+    persistence:
+      enabled: false
+    livenessProbe:
+      enabled: false
+    readinessProbe:
+      enabled: false
```

Comment:
Looks like these probes were actually right. These adjustments should fix the root cause: https://github.com/ipfs/testground/pull/798 Then we can keep the probes enabled, and only merge the persistence disabling.